TMDbLib/Objects/Exceptions/TMDbStatusMessage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace TMDbLib.Objects.Exceptions
+{
+    public class TMDbStatusMessage
+    {
+        [JsonProperty("status_code")]
+        public int StatusCode { get; set; }
+
+        [JsonProperty("status_message")]
+        public string StatusMessage { get; set; }
+    }
```

Comment:
This is the object from TMDb?

Reply:
Yep, an example could be: http://api.themoviedb.org/3/credit/9999999999?api_key=c6b31d1cdad6a56a23f0c913e2482a31

Reply:
Good.